spec/services/hyrax/form_factory_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+RSpec.describe Hyrax::FormFactory do
+  subject(:factory) { described_class.new }
+  let(:model)       { FactoryBot.build(:hyrax_work) }
+
+  describe '.build' do
+    let(:ability)    { :FAKE_ABILITY }
+    let(:controller) { :FAKE_CONTROLLER }
+
+    it 'returns a change set' do
+      expect(factory.build(model, ability, controller)).to be_a Hyrax::ChangeSet
```

Comment:
Would it be out of scope to have named parameters for the `#build` method?

Reply:
Based on our conversation, @no-reply stated that we are conforming to the `#build` method from `Hyrax::WorkFormService`.

See

https://github.com/samvera/hyrax/blob/1b0afbd6749f35ea771f88da47cf4cf32a0a92fa/app/services/hyrax/work_form_service.rb#L3

Reply:
And we're not open to changing that API at this point.